cpp/capi.cpp

```diff
@@ -228,8 +233,8 @@ void guiappSetWindowIcon(const char *path, int pathLen)
 QQmlComponent_ *newComponent(QQmlEngine_ *engine, QObject_ *parent)
 {
     QQmlEngine *qengine = reinterpret_cast<QQmlEngine *>(engine);
-    //QObject *qparent = reinterpret_cast<QObject *>(parent);
```

Comment:
Not a clue. `parent` was never used anyway, but I thought if the code had it, then it should work. I was experimenting with setting the parent to have it destroyed automatically when the engine is destroyed, but that didn't work.